twitter4j-ads/src/twitter4jads/impl/TwitterAdsCampaignApiImpl.java

```diff
@@ -31,8 +31,7 @@
  */
 public class TwitterAdsCampaignApiImpl implements TwitterAdsCampaignApi {
 
-    private static final Integer MAX_REQUEST_PARAMETER_SIZE = 50;
```

Comment:
Any specific reason for doing this? I'm not sure how the 50 number was there before but has there been an update to API docs or best practices?

Reply:
The current max value that Twitter allowed, is up to 200 IDs, so why not use the max value and save API calls?